README.md

```diff
@@ -8,12 +8,11 @@ Laravel MongoDB
 
 This package adds functionalities to the Eloquent model and Query builder for MongoDB, using the original Laravel API. *This library extends the original Laravel classes, so it uses exactly the same methods.*
 
+This package was renamed to `mongodb/laravel-mongodb` because of a transfer of ownership to MongoDB, Inc.
+It is compatible with Laravel 10.x. For older versions of Laravel, please refer to the [old versions](https://packagist.org/packages/jenssegers/mongodb).
```

Comment:
How does this URL differ from https://packagist.org/packages/mongodb/laravel-mongodb? Both show the same tags.

Reply:
Well, nothing except the readme on the homepage will not be updated. I updated link to the old version: https://github.com/mongodb/laravel-mongodb/tree/3.9#laravel-version-compatibility

I found the old versions can be installed for using the new package name even if that's not the name in `composer.json`.